boards/st/steval_stwinbx1/steval_stwinbx1.dts

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+ #include <st/u5/stm32u585Xi.dtsi>
+ #include <st/u5/stm32u585aiixq-pinctrl.dtsi>
+ #include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "STMicroelectronics STEVAL-STWINBX1 Development kit";
+	compatible = "st,steval_stwinbx1";
+
+	chosen {
+		/*
+		 * By default, Zephyr console and shell are assigned to
+		 * USB CDC/ACM. To enable console and shell over UART,
+		 * uncomment the 2 following lines and set the correct
+		 * config in steval_stwinbx1_defconfig.
+		 */
+		/* zephyr,console = &usart2; */
+		/* zephyr,shell-uart = &usart2; */
+		/*
+		 * To enable console and shell over UART,
```

Comment:
I have seen that in some boards, it is suggested to add an overlay file instead of modifying the original dts file (see https://github.com/zephyrproject-rtos/zephyr/blob/4efc03520966a96e1a5dcf5369f75503626c2b42/boards/u-blox/ubx_bmd345eval/doc/index.rst?plain=1#L465) to manage similar needs.

I would consider to use the same approach here and possibly update also sensor tile box pro.

Reply:
Indeed, that would be cleaner. I suspect we have the same on few other boards as well. Let's use this in current PR and open an issue to update other boards.